Sources/Runner/Commands/Runner.swift

```diff
@@ -71,30 +70,37 @@ func runDanger(logger: Logger) throws {
         libArgs += ["-L", libDangerPath] // Link to libDanger inside this folder
         libArgs += ["-I", libDangerPath] // Find libDanger inside this folder
 
-        let importExternalDeps = importsOnly.components(separatedBy: .newlines).filter { $0.hasPrefix("import") && $0.contains("package: ") } // swiftlint:disable:this line_length
+        let importExternalDeps = importsOnly
+            .components(separatedBy: .newlines)
+            .filter { $0.hasPrefix("import") && $0.contains("package: ") } // swiftlint:disable:this line_length
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'l...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 75;
-->

- :warning: SwiftLint rule 'line_length' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)